### PR TITLE
Added sensor name to example file name

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

**Comment by brentru** [comment]
@dherrada  Please include `_simpletest` before the `.py` extension on this example file.

**Comment by brentru** [comment]
@dherrada  Please check that you're providing the correct path in the docs

https://travis-ci.com/adafruit/Adafruit_CircuitPython_VC0706/builds/110796673#L633

**Comment by brentru** [comment]
Docs look good - thanks! Please include `_simpletest` on the example you updated the name of. Then, update the docs to reflect this change

File: docs/examples.rst [hunk 1]
```diff
@@ -3,10 +3,10 @@ Simple test
...
-.. literalinclude:: ../examples/snapshot_internal.py
-    :caption: examples/snapshot_internal.py
+.. literalinclude:: ../examples/vc0706_snapshot_internal.py
+    :caption: examples/vc0706_snapshot_internal.py
     :linenos:
 
-.. literalinclude:: ../examples/snapshot.py
-    :caption: examples/snapshot.py
+.. literalinclude:: ../examples/vc0706_snapshot_simpletest.py
+    :caption: examples/vc0706_snapshot.py
```
Comment (brentru) [inline docs/examples.rst]: Please change this to vc0706_snapshot_simpletest, Line #11

**Comment by dherrada** [comment]
Done

**Comment by brentru** [comment]
Thanks @dherrada, looks good.

**Pull request merged**